modules/data.atmosphere/R/met.process.R

```diff
@@ -350,7 +350,6 @@ met.process <- function(site, input_met, start_date, end_date, model,
                                     site = site, 
                                     start_date = start_date,
                                     end_date = end_date, 
-                                    browndog = browndog, 
```

Comment:
Reminder that there are other Browndog references to be deleted from this file -- I assume those are on your list already, just noting here so we don't forget them

Reply:
I'm eliminating all `Browndog` existences turn by turn. For now, we have eliminated browndog from `convert_input`. Next would be removal from `met.process` and then other instances